shell/platform/android/test/io/flutter/plugin/editing/SpellCheckPluginTest.java

```diff
@@ -209,6 +210,14 @@ public void onGetSentenceSuggestionsResultsWithSuccessAndResultsProperly() {
               new int[] {5})
         });
 
-    verify(mockResult).success(new ArrayList<String>(Arrays.asList("7.11.world\nword\nold")));
+    ArrayList<HashMap<String, Object>> expectedResults = new ArrayList<HashMap<String, Object>>();
+    HashMap<String, Object> expectedResult = new HashMap<String, Object>();
+
+    expectedResult.put("startIndex", 7);
+    expectedResult.put("endIndex", 12);
+    expectedResult.put("suggestions", new ArrayList<String>(Arrays.asList("world", "word", "old")));
```

Comment:
Nit: Would it be beneficial to store these keys as constants somewhere that SpellCheckPlugin.java and this test could both share them?

No need to worry about this is other similar tests don't do this, etc.